gulpfile.js

```diff
@@ -47,6 +49,14 @@ function handleError() {
   });
 }
 
+gulp.task('sitemap', function() {
+  gulp.src('dist/**/*.html')
+    .pipe(sitemap({
+      siteUrl: process.env.teachURL || 'https://teach.webmaker.org'
```

Comment:
do you mind changing this env var to be called `ORIGIN`? That way if we ever add persona etc it can just use that URL... and I'd also like to refer to all env vars in uppercase w/ underscores as a general matter of style.

Can you also document the env var in `README.md`?